docusaurus/docs/develop/path/7_e2e_and_load_tests.md

```diff
@@ -32,21 +32,33 @@ description: End-to-End & Load Tests for PATH
 **Run E2E tests with Docker containers:**
 
 ```bash
-# Shannon E2E tests (local Anvil)
-make test_e2e_evm_shannon
+# Shannon E2E tests with default service IDs
+make test_e2e_evm_shannon_defaults
 
-# Morse E2E tests (real networks)
+# Shannon E2E tests with specified service IDs
+make test_e2e_evm_shannon eth,anvil
+
+# Morse E2E tests with default service IDs
 make test_e2e_evm_morse
+
+# Morse E2E tests with specified service IDs
+make test_e2e_evm_morse F00C,F021
 ```
 
 **Run load tests against existing gateways:**
 
 ```bash
-# Shannon load tests
-make test_load_evm_shannon
+# Shannon load tests with default service IDs
+make test_load_evm_shannon_defaults
+
+# Shannon load tests with specified service IDs
+make test_load_evm_shannon eth,anvil
 
-# Morse load tests
-make test_load_evm_morse
+# Morse load tests with default service IDs
+make test_load_evm_morse_defaults
+
+# Morse load tests with specified service IDs
+make test_load_evm_morse F00C,F021
```

Comment:
Personally, I feel like we should just get rid of the "defaults" and keep targets+examples with specific service IDs only.

Reduce optionality for the user.